src/main.rs

```diff
@@ -41,6 +53,25 @@ enum Commands {
         /// If this path is provided and it exists, the private key is read from this file and used, if it does not exist the private key will be persisted to this location.
         #[clap(long)]
         key: Option<PathBuf>,
+        /// Optional rpc port, defaults to 4919
+        #[clap(long)]
+        rpc_port: Option<u16>,
+    },
+    /// List hashes
+    #[clap(about = "List hashes")]
+    List {
+        /// Optional rpc port, defaults to 4919
+        #[clap(long)]
+        rpc_port: Option<u16>,
+    },
+    /// Add some data to the database.
+    #[clap(about = "Add data from the given path")]
+    Add {
+        /// The path to the file or folder to add.
+        path: PathBuf,
+        /// Optional rpc port, defaults to 4919
+        #[clap(long)]
+        rpc_port: Option<u16>,
```

Comment:
if you use `default_value_t` to specify the default it should automatically document it